.github/workflows/release.yml

```diff
@@ -0,0 +1,60 @@
+---
+name: release
+
+on:
+  release:
+    types: [released]
+
+env:
+  AWS_DEFAULT_REGION: us-east-2
+  BUILD_REGION: us-east-2
+  CURL_CACHE_DIR: $HOME/.cache/curl
+  DEPLOY_REGION_KMS_MAP: "us-east-1:alias/cool/ebs,
+                          us-east-2:alias/cool/ebs,
+                          us-west-1:alias/cool/ebs,
+                          us-west-2:alias/cool/ebs"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: cisagov/setup-env-github-action@develop
+      - uses: actions/checkout@v1
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Setup cache # GitHub support coming soon
+        run: mkdir -p $CURL_CACHE_DIR
+      - name: Install Packer
+        run: |
+          PACKER_ZIP="packer_${PACKER_VERSION}_linux_amd64.zip"
+          curl --output "${CURL_CACHE_DIR}/${PACKER_ZIP}" \
+            --time-cond "${CURL_CACHE_DIR}/${PACKER_ZIP}" --location \
+            "https://releases.hashicorp.com/packer/${PACKER_VERSION}/${PACKER_ZIP}"
+          sudo unzip -o -d /usr/local/bin "${CURL_CACHE_DIR}/${PACKER_ZIP}"
+      - name: Install Terraform
```

Comment:
Could the install of packer and terraform be broken out into a separate repo a la what you did with [cisagov/setup-env-github-action](https://github.com/cisagov/setup-env-github-action)?

Seems maybe the Ansible role stuff could possibly be broken out as well, taking the list of role names as an input.